Source/WebCore/platform/mediastream/mac/BaseAudioSharedUnit.cpp

```diff
@@ -60,6 +64,8 @@ void BaseAudioSharedUnit::removeClient(CoreAudioCaptureSource& client)
 
 void BaseAudioSharedUnit::clearClients()
 {
+    RELEASE_LOG_ERROR(WebRTC, "BaseAudioSharedUnit::clearClients %p", this);
```

Comment:
Ditto

Reply:
This should be removed, I did it in the new patch